src/styles/blocks/button.scss

```diff
@@ -0,0 +1,45 @@
+.button {
+  &__link {
+    box-sizing: border-box;
+    display: block;
+    font-size: 16px;
+    line-height: 27px;
+    text-align: center;
+    text-decoration: none;
+    color: #FFF;
+    padding: 15px 48px;
+    background-color: $main-text-color;
+    border: 2px solid $main-text-color;
+    border-radius: 8px;
+    transition: all 0.3s;
+
+    &:hover {
+      color: $main-text-color;
+      border-color: #F0F1F4;
+      background-color: #F8F8FA;
```

Comment:
Move all colors to a variable to improve readability and remove magic colors

Also this will prepare you to work with some kind of design system where you have a specific colors not a random one